index.js

```diff
@@ -38,7 +38,7 @@ got.stream(cfg.wolke)
 		// Compute average positions by time
 		const centers = normalized[line] = []
 		for (let i = 0; i < maxLength; i++) {
-			const position = center(trips.map((trip) => trip[i]))
+			const position = center(trips.map((trip) => {lat: trip[i].latitude, lon: trip[i].longitude}))
```

Comment:
Did you test this? I think it won't work because, in arrow functions, if you want to return an object, you have to wrap it inside parentheses (e.g. `({foo: 'bar'})`).